app/models/discovery_attribute_set.rb

```diff
@@ -1,5 +1,6 @@
 class DiscoveryAttributeSet < ApplicationRecord
   belongs_to :host, :class_name => "Host::Discovered", :foreign_key => :host_id
+  audited :associated_with => :host
```

Comment:
This can theoretically generate a lot of noise if there is some host (buggy for example) which reports change in memory or cpu count or disks amount on every request. Discovered hosts sends facts every 10 minutes or so by default. Just a warning, if you have some 2nd tier audits (things which users can turn off), this is a good candidate.